mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -606,13 +672,28 @@
     </CAbilMorph>
     <CAbilMorph id="MorphToLurker">
         <Flags index="IgnoreFacing" value="1"/>
+        <Flags index="WaitUntilStopped" value="0"/>
+        <InfoArray index="0" RandomDelayMax="0"/>
```

Comment:
**undocumented**: Morphing to a Lurker's random delay reduced from [0,0.5] to [0,0]

Was this necessary for the smart command cancellation? Why is this undocumented?